OntologyMapper/src/OntologyMapper/OntologyMapping.cs

```diff
@@ -0,0 +1,167 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="OntologyMapping.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.SmartFacilities.OntologyMapper
```

Comment:
This is starting off with SmartFacilities, but the OntologyMapper could be used for all sorts of verticals. Since its DTDL -> DTDL I wouldn't expect there shouldn't be any domain specific logic.

Reply:
Not sure what namespace to put this in at this point. And what are the Microsoft guideline for it. We will likely move it before creating the nuget